src/backend/gl/Cargo.toml

```diff
@@ -17,29 +17,25 @@ build = "build.rs"
 name = "gfx_backend_gl"
 
 [features]
-#default = ["wgl", "surfman", "surfman-x11"]
-default = []
-wgl = ["winapi"]
-#surfman-x11 = ["surfman/sm-x11"]
+default = ["surfman"]
+surfman-x11 = ["surfman/sm-x11"]
 
 [dependencies]
 arrayvec = "0.5"
 bitflags = "1"
-log = { version = "0.4" }
-gfx-hal = { path = "../../hal", version = "0.6" }
+log = "0.4"
+hal = { path = "../../hal", version = "0.6", package = "gfx-hal" }
 auxil = { path = "../../auxil/auxil", version = "0.7", package = "gfx-auxil", features = ["spirv_cross"] }
 smallvec = "1.0"
-glow = "0.4"
+glow = "0.6.1"
 parking_lot = "0.11"
 spirv_cross = { version = "0.22", features = ["glsl"] }
 lazy_static = "1"
 raw-window-handle = "0.3"
-# disable this for now, the versions of glutin are jumping too fast
-#glutin = { version = "0.24", optional = true }
 
-#[target.'cfg(all(unix, not(target_os = "ios")))'.dependencies]
+[target.'cfg(all(unix, not(target_os = "ios")))'.dependencies]
 # TODO: Update to released version when it comes out
-#surfman = { git = "https://github.com/servo/surfman.git", rev = "41ac1ee", features = ["sm-raw-window-handle"], optional = true }
+surfman = { git = "https://github.com/servo/surfman.git", rev = "41ac1ee", features = ["sm-raw-window-handle"], optional = true }
```

Comment:
aah, I forgot about this one. We can't (and shouldn't) proceed with these changes while Surfman is not published, since we can't publish anyway until then

Reply:
wait, can't we just use `surfman-0.3` here from crates?

Reply:
Umm. It looks like it actually. That may have been old. I'll fix that.